_posts/2021-04-21-frequency-band-control-in-unetstack.md

```diff
@@ -0,0 +1,199 @@
+tags: [howto, unetstack, modems, phy, frequency, band, fdma]
+---
+
+## The basics
+
+An underwater acoustic modem essentially sends out sound signals underwater to transmit information. The typical transmit processing takes in user's binary data and after error correction encoding, modulates a carrier according to some modulation scheme such as OFDM or FHBFSK, the details of which the reader can understand from other sources. 
+
+For standard medium frequency Subnero modems, the fundamental centre frequency `C=24kHz`.
+The baseband complex signals are clocked out via the DAC (Digital to Analog Converter) at the rate of `R=24000 samples/s`, which translates to a total bandwidth of `B=24kHz` (we shall omit the details of wireless communication theory in this blog). So the band goes from 12kHz to 36kHz.
+
+For Unet audio, the centre frequency `C=12kHz`, the complex baseband rate `R=12000` and the total bandwidth `B=12kHz`. So it goes from 6kHz to 18kHz.
+
+For the illustrations in this blog we shall use Unet audio mainly. Change `C`, `R` and `B` according to the modem applicable to the reader.
+
+The reader should install Unet and Unet audio and try out some basic examples to make sure it is working. The following sections can be tried out in Unet audio.
+
+### The need to control centre frequency and bandwidth 
+
+Acoustic modems transmit physical sound waves via a transducer, typically a piezo electric device. Such sound emitters have an ideal resonance frequency F and a Q factor of the order of 0.3. This means that the efficient region of the frequencies it can transmit centres around the resonance frequency with a bandwidth of about 0.3 x F. Q may be higher or lower depending on the exact transducer.
+
+The transducer in medium range Subnero modem has a transducer with a resonance close to 26kHz << check>>. The recommended default bandwidth is B=12kHz which is half of the computationally possible bandwidth of 24kHz. Thus an effective transmit modulation scheme can be set such that the centre `C=26kHz` and `B=12kHz`.
+
+Another reason may be due to acoustic underwater channel characteristics. A most basic property is that higher frequencies get absorbed more and in general have lower transmission range. For this or other reasons related to the channel, the modem user may want to customize the frequencies transmitted by the modem. 
+
+Multiple Access is another case where users may want to control the band used in each modem. One common multiple access method is Frequency Domain Multiple Access (FDMA). In this scheme, different modems or modem pairs may be allocated a different frequency band for transmission to avoid collision. Simultaneous transmission-reception by two or more different and colocated modem pairs are possible using FDMA. 
+
+Sometimes, there may be a frequency band the user may want to exclude from usage, due to it being occupied by other sources, noise etc. 
+
+## Understanding baseband signal generation, transmission 
```

Comment:
Open question: Will this section add to the user's understanding or scare away end-users who just want to change parameters and don't want to understand BPSK?

A middle ground may be a lead in statement which explains the purpose of this section, and that this section may be safely skipped by a user who only wants to adjust FH-BFSK or OFDM parameters of their modem.